### ISSUE TITLE
Iframe with X-Frame-Options set to 'sameorigin' in response blocked by browser.

### ISSUE THREAD

**Issue by Haris-Jamil** [nwjs/nw.js#7167]
Thanks for reporting an issue! Spend 60 seconds reading this...

1. Before reporting, try reproducing against the latest version of NWJS.
2. Issue tracker is only for bugs and feature requests, in English only.
  
  Anything else? Use our mailing list for questions: 
  https://groups.google.com/forum/#!forum/nwjs-general

======== TEMPLATE BUG FORM ========

NWJS Version : 36.1
Operating System : Windows 7

### Expected behavior
Should allow rendering of webpage inside iframe

### Actual behavior
Showing broken page error inside iframe

I am creating an iframe in my NW.js app like this

```
var iFrame = document.createElement('iframe');
iFrame.setAttribute('id',frameId);
iFrame.setAttribute('style','display:block');
iFrame.setAttribute('sandbox','allow-top-navigation allow-scripts allow-forms allow-same-origin');
iFrame.style.width = 100 +'%';
iFrame.style.height = 100 +'%';
dom.appendChild(iFrame);
```
i have set node-remote and chromium-args parameters in my package.json with following values

```
"node-remote": [ "*://*" ],
"chromium-args": "--disable-web-security --enable-html-imports --allow-file-access",
```
lets say the src of my iframe is https://www.abcSomepage.com but it have X-Frame-Options set to sameorigin in it's response headers, due to which app is showing a broken page in iframe,

**Comment by rogerwang** [comment]
close as duplicate of #7049
